content/applications/finance/expenses/reimburse.rst

```diff
@@ -0,0 +1,134 @@
+===================
+Reimburse employees
+===================
+
+After an expense report is :ref:`posted to an accounting journal <expenses/post>`, the next step is
+to reimburse the employee. Just like approving and posting expenses, employees can be reimbursed in
+two ways: either with cash, check, or direct deposit (:ref:`individually
+<expenses/reimburse-single>` or :ref:`in bulk <expenses/reimburse-bulk>`), or :ref:`reimbursed in a
+payslip <expenses/reimburse-payslip>`.
+
+Settings
+========
+
+Reimbursements can be paid via paycheck, check, cash, or bank transfer. To set up payment options,
+first configure the various settings by navigating to :menuselection:`Expenses app -->
+Configuration --> Settings`.
+
+To reimburse employees for expenses :ref:`in their paychecks <expenses/reimburse-payslip>`, activate
+the :guilabel:`Reimburse in Payslip` option in the :guilabel:`Expenses` section.
+
+Next, set how payments are made in the :guilabel:`Accounting` section. Click the drop-down menu
+under :guilabel:`Payment Methods`, and select the desired payment option. Default options include
+paying by :guilabel:`Cash`, :guilabel:`Check`, :guilabel:`NACHA`, and others. Leaving this field
+blank allows for **all** available payment options to be used.
+
+.. _expenses/reimburse-single:
+
+Reimburse individually
+======================
+
+To reimburse an individual expense report, first navigate to :menuselection:`Expenses app -->
+Expense Reports --> Reports To Pay`. All expense reports are presented in a default list view.
```

Comment:
I feel like I am going crazy, because I definitely saw that in one of the DB's I was working with! I just rechecked the runbots for today, and my own test DB, and you are correct- I honestly don't know where that came from, except I know I definitely saw it. I feel like I said I saw a ghost =D